dvc/main.py

```diff
@@ -53,6 +54,15 @@ def main(argv=None):
         ret = 253
     except DvcParserError:
         ret = 254
+    except UnicodeError:
+        if is_py2:
+            logger.exception(
+                "unicode is not fully supported in DVC for Python 2.\n"
+                "Python 2.7 will not be maintained past 2020.\n"
+                "Please upgrade to Python 3 to work correctly with unicode.\n"
+                "Exception"
```

Comment:
No need for this "Exception"

Reply:
I thought it looked reasonable: `Exception - 'ascii' codec ...` instead of `Please upgrade to Python 3 - 'ascii' codec ...`